readme.md

```diff
@@ -30,6 +42,10 @@ machine.start(function (err) {
 })
 ```
 
+## `Machine.list((err, result) => ..)`
+
+List all machines.

```

Comment:
Would be nice if the docs mention that `results` is an array. Perhaps also include a link to [docker-machine ls](https://docs.docker.com/machine/reference/ls/), so that people know what fields to expect.